packages/sync/src/connect-webrtc.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+// @ts-ignore
+import { WebrtcProvider } from 'y-webrtc';
+
+/** @typedef {import('./types').ObjectType} ObjectType */
+/** @typedef {import('./types').ObjectID} ObjectID */
+/** @typedef {import('./types').CRDTDoc} CRDTDoc */
+
+/**
+ * Connect function to the IndexedDB persistence provider.
```

Comment:
I think the comment here is just a copy of the IndexDB comment

```suggestion
 * Connect function to the WebRTC provider.
```